src/components/ArcadeMode.jsx

```
@@ -0,0 +1,548 @@
+                    className={`progress-connector ${level < (tournamentManager?.currentOpponent || 1) ? 'completed' : ''}`}
+                  />
+                )}
+              </>
+            ))}
+          </div>
+        </div>
+      )
+
+    case 'advancingTransition':
+      const nextOpponent = tournamentManager?.getCurrentOpponent()
+      const currentLevel = tournamentManager?.currentOpponent || 1
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)